guides/deploying-to-digital-ocean-with-ansible.md

```diff
@@ -0,0 +1,167 @@
+## At a Glance
+
+This deployment guide's purpose is to provide a simple and easy way to deploy Open Commerce for evaluation purposes or small deployments. This guide is not meant to generate an enterprise production grade deployment, rather Docker Compose is used to manage containers which is not suitable for production deployments.
+
+> Note: It is important to understand that Open Commerce cannot provide support for your deployment. If you need 
+> help you can reach out to the community on our [Discord Server](https://discord.gg/Bwm63tBcQY)
+
+## What you need
+
+- A Linux host with at least 2GB of RAM, this guide uses a DigitalOcean droplet
+- A registered domain
+- A DNS manager that supports Certification Authority Authorization (CCA) records, such as Digital Ocean
+- Some familiarity with [Traefik](https://github.com/containous/traefik/)
+
+## Understand what the deployment will look like
+
+**Open Commerce Services Overview**
+
+- **Open Commerce GraphQL API**:
+  The [Open Commerce GraphQL API](https://github.com/reactioncommerce/reaction) service provides the GraphQL interface 
+  to the Open Commerce core functionality.
+- **Storefront**:
+  The [Example storefront](https://github.com/reactioncommerce/example-storefront) service provides the public facing storefront interface that customers will interact with.
+- **Open Commerce Admin**:
+  The [Admin](https://github.com/reactioncommerce/reaction-admin) service is a Meteor application that provides the UI to manage products, orders etc.
+
+Open Commerce services will be exposed to the public using [Traefik](https://github.com/containous/traefik/), which 
+is a cloud native router. Traefik will act as a reverse proxy that will route traffic to Docker containers. As stated above, you will need a registered domain to complete this step, as it will be necessary to manage DNS records for it.
+
+This guide will use the following subdomains, where `example.com` will need to substitute it with your domain:
```

Comment:
This guide will use the following subdomains, where `example.com` will need to be substituted with your domain: